### DISCUSSION TITLE
关于封面信息的排版问题

### DISCUSSION THREAD

**Discussion by zyklovecoding** [whutug/whu-thesis#301] [Q&A] [answered]
您好请问该如何更改冒号右侧的信息为下图所示的左对齐呢，我尝试更改了whu-thesis.cls中的样式，但由于我更改的位置不对，因此最终没有效果，感谢您的回复

**Comment by SwitWu** [comment, marked as answer]
https://github.com/whutug/whu-thesis/blob/a03ab5a275c84afdd4085d8544bcdf8b313e58b7/whu-thesis.cls#L1455

个人信息用了分散对齐的盒子，将这行代码改为 `\l__whu_tmpb_tl` 即可
